permissions_engine/calculate.rego

```diff
@@ -54,14 +54,24 @@ team_readable_programs[p] := output if {
 }
 
 # user can read programs that are either team-readable or user-readable
-readable_programs := object.keys(object.union(team_readable_programs, user_readable_programs))
+readable_programs := all_programs if {
+	user_key in site_roles.curator
+}
+
+else := object.keys(object.union(team_readable_programs, user_readable_programs))
 
 # user can curate programs that list the user as a program curator
-curateable_programs[p] if {
+program_curateable_programs[p] if {
 	some p in all_programs
 	user_key in program_auths[p].program_curators
 }
 
+curateable_programs := all_programs if {
```

Comment:
`program_curateable_programs` is really confusing to me -- maybe if this variable was renamed to `site_curateable_programs` to mirror the other variable it'd be easier to understand?

Reply:
It's not the best name, but I needed to make the original curateable_programs be able to be either all_programs or this other thing. I use curateable_programs for other things, so I didn't want to change it. program_curateable_programs shouldn't show up in the decision logs or anything.

Reply:
Maybe add a comment above this one explaining that it's for site_curators?